systemtests/tests/restore/testrunner-restore-fileregex

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+set -e
+set -o pipefail
+set -u
+#
+# Restore files and filter them with FileRegex
+#
+TestName="$(basename "$(pwd)")"
+export TestName
+
+#shellcheck source=../environment.in
+. ./environment
+
+#shellcheck source=../scripts/functions
+. "${rscripts}"/functions
+#shellcheck source=functions
+. functions
+
+# Use own restore directory to ensure it is always
+# clean before restore
+RestoreDirectory="${tmp}/fileregex-restores"
+
+start_test
+
+fileregex=".*/ot[h]er-.*"
+
+matched_files=$(find "$BackupDirectory/" -type f -regex "$fileregex")
```

Comment:
This does not work if paths contain whitespaces!
An alternative could be
```
mapfile -d $'\0' matched_files < <(find "$BackupDirectory/" -type f \
					-regex "$fileregex" -print0)
```

Reply:
Done